ci/azure-pipelines.yml

```diff
@@ -0,0 +1,20 @@
+jobs:
+  - job: UnitTest
+    pool:
+      vmImage: 'VS2017-Win2016'
+    steps:
+      - template: windows-dependencies.yml
+        parameters:
+          rustup_toolchain: '1.34.2-x86_64-pc-windows-msvc'
+      - script: make test
+        displayName: Run unit tests
+  - job: IntegrationTest
+    condition: eq(variables['Build.Reason'], 'PullRequest')
```

Comment:
The condition is inversed. We only run unit tests for PR, and integration for branch.

Reply:
Better move the file into `devtools`. It seems Azure Pipeline allows us to choose the yml file path.